### PR TITLE
Add end screen

### PR DESCRIPTION
**Pull request by BWalusiak**
Add missing textures, add nuke bomb, add end screen WIP

### CODE REVIEW

File: client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java [hunk 1]
```diff
@@ -116,9 +119,11 @@ public static int getBombVariant(BombEntity bombEntity) {
         ItemType type = bombEntity.getBombItemType();
         switch (type) {
             case SMALL_BOMB:
-                return BombSprite.VARIANT_SMALL_RED; // TODO: Check the player texture variant
+                return BombSprite.VARIANT_SMALL_RED;
```
Comment (RouNNdeL) [inline client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java]: Why was this TODO removed? The feature is still missing.

**Review by RouNNdeL — CHANGES_REQUESTED** [review]

File: client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java (hunk 1 above)
Comment (BWalusiak) [reply client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java]: This feature was deemed as an inefficient use of company resources at the 13.06.2020 board meeting, thus making this TODO unnecessary.

Comment (RouNNdeL) [reply client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java]: Well in that case, this method is useless, as is the Bomb's playerTextureIndex property and variant constants, all of which should be removed.

Comment (BWalusiak) [reply client/core/src/com/cyberbot/bomberman/sprites/GraphicsFactory.java]: Removed

**Review by RouNNdeL — APPROVED** [review]

**Pull request merged**